BGRSCRPT/iocBoot/iocBGRSCRPT-IOC-01/config.xml

```diff
@@ -4,6 +4,7 @@
   <ioc_desc>Background Script</ioc_desc>
   <ioc_details>IOC runs a background script from configurations area</ioc_details>
   <macros>
+    <macro name="SCRIPT_PATH" pattern="^.*$" description="Path to the Python background script." hasDefault="NO"/>
```

Comment:
```suggestion
    <macro name="SCRIPT_PATH" pattern="^.*$" description="Path to the Python background script." hasDefault="YES"/>
```

This does have a default, doesn't it? The IOC boot script looks to say that if this isn't defined it defaults to `%ICPINSTSCRIPTROOT%/background_script.py`